v3/plugins/scalameta/metals/1.7.2/meta.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+publisher: scalameta
+name: metals
+version: 1.7.2
+type: VS Code extension
+displayName: Scala (Metals)
+title: Scala language server with rich IDE features
+description: This plug-in provides support for Scala development via the Metals language server.
+icon: https://rawcdn.githack.com/OlegIlyenko/scala-icon/c453c6f67dfcc1dbf0ca1e437d32d04f1d710983/scala-icon.svg
+repository: https://github.com/scalameta/metals-vscode
+category: Language
+firstPublicationDate: "2020-01-24"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-java:8-0cfbacb"
+      name: vscode-metals
+      memoryLimit: "1500Mi"
+      volumes:
+      - mountPath: "/home/theia/.m2"
+        name: m2
+  extensions:
+    - https://github.com/sideeffffect/metals-vscode/releases/download/1.7.2/scalameta.metals-1.7.2.vsix
```

Comment:
We generally want to avoid pulling vsixes from forked repos. I'd prefer a more stable vsix source.

Reply:
I will talk to the extension developers after we test that it's capable of running in Che (https://github.com/eclipse/che/issues/15820)

Reply:
Thanks @sideeffffect for your contributions!